resources/js/stores/useCart.js

```diff
@@ -55,19 +44,27 @@ export const clearAddresses = async function () {
 
 export const cart = computed({
     get() {
-        if (!cartStorage.value?.entity_id && mask.value) {
-            refresh()
+        if (!cartStorage.value?.id && mask.value) {
+            // TODO: Check as we don't want this all the time.
```

Comment:
I think we do want this all the time, since this only gets executed when we do not have a cart but do have a mask.
Which means either we **should** have a cart or we **shouldn't** have a mask.

These cases get sorted out by the refresh function.
 - If we have no cart but we have a valid mask, we retrieve the cart
 - If we have no cart and we have an invalid mask, the mask gets removed with the notification that the cart has expired.
 - If we have a cart and a mask this will not get triggered